vss-extension.json

```diff
@@ -117,42 +117,76 @@
                     "displayName": "Marketplace URL",
                     "value": "https://marketplace.visualstudio.com"
                 },
-                "authenticationSchemes": [
+              "authenticationSchemes": [
+                {
+                  "type": "ms.vss-endpoint.endpoint-auth-scheme-token",
```

Comment:
In that case I propose 2 endpoints. One named Visual Studio Marketplace,
one called TFS Marketplace.

Then put in a radio button or dropdown on the connected service to pick
which one using visible rule.